tests/components/twitch/conftest.py

```diff
@@ -0,0 +1,114 @@
+"""Configure tests for the Twitch integration."""
+from collections.abc import Awaitable, Callable, Generator
+import time
+from unittest.mock import AsyncMock, patch
+
+import pytest
+
+from homeassistant.components.application_credentials import (
+    ClientCredential,
+    async_import_client_credential,
+)
+from homeassistant.components.twitch.const import DOMAIN
+from homeassistant.core import HomeAssistant
+from homeassistant.setup import async_setup_component
+
+from tests.common import MockConfigEntry
+from tests.components.twitch import TwitchMock
+from tests.test_util.aiohttp import AiohttpClientMocker
+
+ComponentSetup = Callable[[TwitchMock | None], Awaitable[None]]
+
+CLIENT_ID = "1234"
+CLIENT_SECRET = "5678"
+TITLE = "Test"
+SCOPES = ["user:read:subscriptions", "user:read:follows"]
+
+TWITCH_TOKEN_URI = "https://id.twitch.tv/oauth2/token"
+TWITCH_AUTHORIZE_URI = "https://id.twitch.tv/oauth2/authorize"
```

Comment:
Why aren't you using the Oauth variables from the const?

> OAUTH2_AUTHORIZE = "https://id.twitch.tv/oauth2/authorize"
OAUTH2_TOKEN = "https://id.twitch.tv/oauth2/token"